### PR TITLE
navbar login-page logo better-code responsive

### LINKED ISSUE #74: Improve the current UI
the current UI of the admin panel is not according to the figma designs. Someone can take this issue to improve the UI

link of the figma UI https://www.figma.com/file/KMO0QXuppBfxH9PyuI5mn2/Talawa-Admin?node-id=1%3A4

### LINKED ISSUE #77: Use button instead of input field
btn
The login button in the develop branch on the login page is made using the input field.
This shows inconsistency in code and using input field for the button is wrong according to new design rules.

@palisadoes sir would like to work on this issue.

**Additional details**
GitHub Externship : 19-05_Sin666_tfa_262

### LINKED ISSUE #78: Add appropriate spacing for better UX
All the components should have a uniform padding margin, the inconsistency will lead to bad UI/UX.

Further, the login page is not placed on the page properly. It is in the corner leading to not so good display.

Many of the components fail to follow the same and are different from the Figma design suggested.

@palisadoes sir would like to work on this issue.

**Additional details**
GitHub Externship : 19-05_Sin666_tfa_262

### LINKED ISSUE #79: Make website responsive
The admin portal is not responsive on any screen sizes. This is a problem as the admin might access it through different screens and variation in UI would not result in a good user experience.

@palisadoes sir would like to work on this issue.

res

**Additional details**
GitHub Externship : 19-05_Sin666_tfa_262

### PR DESCRIPTION
**Pull request by ssaumyaa7**
**What kind of change does this PR introduce?**
1. Created responsive NavBar
2. Changed logo and favicon
3. Edited login page according to Figma design
4. Made pages responsive for all screen sizes
5. Changed code according to new design rules
6. Deleted boilerplate files (not used for our repo)
7. Better UI/UX

**Issue Number:**

Issues fixed 👍🏻 
Fixes #77  - created button 
Fixes #78  - added appropriate spacing
Fixes #79  - responsiveness added
Fixes #74  - Improved UI/UX

**Summary**
In crux, created better UI/UX with react coding according to new rules.

**Additional details**
GitHub Externship : 19-05_Sin666_tfa_262

@DangaRanga sir, @palisadoes sir, @Sagar2366 sir,
Please review and merge.

### CODE REVIEW

**Comment by yasharth291** [comment]
Hey @ssaumyaa7  I don't think #79 is fixed by this

**Comment by yasharth291** [comment]
Could you please add screen shots of the designs

File: src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import styles from './Navbar.module.css';
+import Navbar from 'react-bootstrap/Navbar';
 import web from 'assets/talawa-logo-lite-200x200.png';
```
Comment (yasharth291) [inline src/components/Navbar/Navbar.tsx:4]: Change the name of the web to logo

File: src/screens/LoginPage/LoginPage.tsx [hunk 2]
```diff
@@ -53,17 +54,14 @@ function LoginPage(): JSX.Element {
                 }}
                 required
               />
-              <input
-                type="submit"
-                className="btn"
-                value="Login"
-                onClick={login_link}
-              />
+              <button type="submit" className={styles.loginbtn} value="Login">
```
Comment (yasharth291) [inline src/screens/LoginPage/LoginPage.tsx:57]: see the new code it should go to homepage on clicking

File: src/screens/LoginPage/LoginPage.tsx [hunk 3]
```diff
@@ -53,17 +54,14 @@ function LoginPage(): JSX.Element {
...
+                Login
+              </button>
             </form>
           </div>
         </section>
       </section>
-    </>
+    </div>
```
Comment (yasharth291) [inline src/screens/LoginPage/LoginPage.tsx:64]: don't close it in whole </div> either move the </> block below div

**Review by yasharth291 — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navbar.tsx (hunk 1 above)
Comment (ssaumyaa7) [reply src/components/Navbar/Navbar.tsx:4]: @yasharth291 you put this name so I decided not to change

File: src/screens/LoginPage/LoginPage.tsx (hunk 2 above)
Comment (ssaumyaa7) [reply src/screens/LoginPage/LoginPage.tsx:57]: @yasharth291 again you worked on this part so I'll check

File: src/components/Navbar/Navbar.tsx (hunk 1 above)
Comment (yasharth291) [reply src/components/Navbar/Navbar.tsx:4]: please change now

File: src/screens/LoginPage/LoginPage.tsx (hunk 3 above)
Comment (ssaumyaa7) [reply src/screens/LoginPage/LoginPage.tsx:64]: okay

File: src/screens/LoginPage/LoginPage.tsx (hunk 2 above)
Comment (yasharth291) [reply src/screens/LoginPage/LoginPage.tsx:57]: it was in the new code merged

Comment (ssaumyaa7) [reply src/screens/LoginPage/LoginPage.tsx:57]: seen, please verify at your end

Comment (yasharth291) [reply src/screens/LoginPage/LoginPage.tsx:57]: Capture

see this

Comment (yasharth291) [reply src/screens/LoginPage/LoginPage.tsx:57]: you have to make the button clickable

**Review by palisadoes — CHANGES_REQUESTED** [review]
@ssaumyaa7 please edit your comments so that it you have a separate `Fixes`  statement for each issue. That's the only way the auto closing will work.

**Review by ssaumyaa7 — COMMENTED** [review]
changes added

File: src/screens/LoginPage/LoginPage.tsx (hunk 3 above)
Comment (yasharth291) [reply src/screens/LoginPage/LoginPage.tsx:64]: who is not recommending it 

i just checked on the official website  of react it is there

Comment (yasharth291) [reply src/screens/LoginPage/LoginPage.tsx:64]: https://reactjs.org/docs/fragments.html#short-syntax

Comment (yasharth291) [reply src/screens/LoginPage/LoginPage.tsx]: it is the newer way to write the fragment

File: src/components/Navbar/Navbar.tsx (hunk 1 above)
Comment (DangaRanga) [inline src/components/Navbar/Navbar.tsx:4]: Can you change this from `web` to `TalawaLogo` or `Logo`?

I missed it in my last review
```suggestion
import Logo from 'assets/talawa-logo-lite-200x200.png';
```

File: src/components/Navbar/Navbar.tsx [hunk 4]
```diff
@@ -1,18 +1,21 @@
...
+      <Navbar bg="">
+        <Navbar.Brand>
+          <a className={styles.logo} href="/">
+            <img src={web} />
+            <strong>Talawa</strong>
+            <strong>Admin</strong>
+          </a>
+        </Navbar.Brand>
+      </Navbar>
+    </div>
```
Comment (DangaRanga) [inline src/components/Navbar/Navbar.tsx:17]: For all of the components so far, React fragments, were used for wrapping the components, so please make use of them to ensure that there is consistency
```suggestion
    <>
      <Navbar bg="">
        <Navbar.Brand>
          <a className={styles.logo} href="/">
            <img src={web} />
            <strong>Talawa</strong>
            <strong>Admin</strong>
          </a>
        </Navbar.Brand>
      </Navbar>
    </>
```

File: src/screens/LoginPage/LoginPage.module.css [hunk 5]
```diff
@@ -1,21 +1,20 @@
...
-  grid-template-columns: 1fr 1fr;
 }
 
 .form_wrapper {
-  margin-top: 8em;
-  margin-left: 35em;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```
Comment (DangaRanga) [inline src/screens/LoginPage/LoginPage.module.css:9]: Absolute positioning may not be necessary here, as you are already using flexbox to position this element in the center of the page.

But if it is, please let me know.

**Review by DangaRanga — CHANGES_REQUESTED** [review]
Good job so far, just a few minor changes are to be made.

1. The form seems to be a bit too wide here. Please consider decreasing the width of the form to around 30% or less so that it's just like the design.

image

2. The unit tests seem to be failing after the changes you've made. If we are going to continue with React-Bootstrap for the component library, we need to ensure that the components are written can have and do have unit tests written for them

File: src/components/Navbar/Navbar.tsx (hunk 1 above)
Comment (ssaumyaa7) [reply src/components/Navbar/Navbar.tsx:4]: @DangaRanga Noted

File: src/components/Navbar/Navbar.tsx (hunk 4 above)
Comment (ssaumyaa7) [reply src/components/Navbar/Navbar.tsx:17]: @DangaRanga On it !

File: src/screens/LoginPage/LoginPage.module.css (hunk 5 above)
Comment (ssaumyaa7) [reply src/screens/LoginPage/LoginPage.module.css:9]: @DangaRanga sir, without absolute positioning this issue arises 
issue

**Review by ssaumyaa7 — COMMENTED** [review]
done

**Comment by ssaumyaa7** [comment]
@DangaRanga sir, I have resolved and made changes.
Please review and merge.

**Review by DangaRanga — APPROVED** [review]
Thanks much for the changes

**Comment by ssaumyaa7** [comment]
@xsh2047  sir, @palisadoes  air, please review and merge

@DangaRanga sir approved of the changes

**Comment by ssaumyaa7** [comment]
> @ssaumyaa7 please edit your comments so that it you have a separate `Fixes` statement for each issue. That's the only way the auto closing will work.

I have done that now

**Comment by ssaumyaa7** [comment]
Please review now

**Review by palisadoes — APPROVED** [review]

**Pull request merged**